Tools/Scripts/webkitpy/api_tests/manager.py

```diff
@@ -151,11 +151,8 @@ def _print_tests_result_with_status(self, status, runner):
                 self._stream.writeln('')
 
     def _initialize_devices(self):
-        # FIXME: Remove the special-case for iOS Simulator once https://bugs.webkit.org/show_bug.cgi?id=256806 is resolved
-        if self._port.port_name in ('iphone-simulator', 'ios-simulator'):
-            SimulatedDeviceManager.initialize_devices(DeviceRequest(DeviceType(hardware_family='iPhone', hardware_type='SE (3rd generation)'), allow_incomplete_match=True), self.host, simulator_ui=False)
-        elif 'simulator' in self._port.port_name:
-            SimulatedDeviceManager.initialize_devices(DeviceRequest(self._port.supported_device_types()[0], allow_incomplete_match=True), self.host, simulator_ui=False)
+        if 'simulator' in self._port.port_name:
+            SimulatedDeviceManager.initialize_devices(DeviceRequest(self._port.DEVICE_TYPE, allow_incomplete_match=True), self.host, simulator_ui=False)
```

Comment:
We really need to retain the original `self._port.supported_device_types()[0]`.

`DEVICE_TYPE` isn't guaranteed to be fully qualified, where as `supported_device_types()` is

Reply:
Gotcha. Changing it back now.